src/redux/greetingSlice.js

```diff
@@ -0,0 +1,47 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const URL_API = 'http://127.0.0.1:3000/greetings/random';
+
+export const fetchRandomGreeting = createAsyncThunk('greetings/fetchRandomGreeting', async () => {
+  try {
+    const response = await fetch(URL_API, {
+      headers: {
+        method: 'GET',
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const greeting = await response.json();
+    return greeting;
```

Comment:
- [ ] Great job so far. However, there's one small thing you need to work on. Your frontend app is facing resource from different origin issue. That's open your app from a new browser different from the one on which the app was built. Kindly look into this [article](https://www.stackhawk.com/blog/react-cors-guide-what-it-is-and-how-to-enable-it/) to get some insight about `cors policy`.

### Page Display

![cors-error](https://github.com/tonnytech/hello-react-front-end/assets/26597432/8cdb9c52-62a9-4066-97a2-8ad6ed9886f0)